apb-base/files/usr/bin/entrypoint.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -ex
```

Comment:
How come we are taking this out?

Reply:
-set -ex

I think your right, I can leave the -x in place without a problem. The `-e` needs to go because we do not want to exit immediately when the ansible-playbook fail anymore. We want to capture their exit code, still attempt to serve test results, and then exit, at least that is my thinking.